tools/confix/file.go

```diff
@@ -4,18 +4,24 @@ import (
 	"embed"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/creachadair/tomledit"
 )
 
 //go:embed data
 var data embed.FS
 
-// LoadConfig loads and parses the TOML document from confix data
-func LoadLocalConfig(name string) (*tomledit.Document, error) {
-	f, err := data.Open(fmt.Sprintf("data/%s-app.toml", name))
+// LoadLocalConfig loads and parses the TOML document from confix data
+func LoadLocalConfig(name string, configType string) (*tomledit.Document, error) {
+	fileName, err := getFileName(name, configType)
 	if err != nil {
-		panic(fmt.Errorf("failed to read file: %w. This file should have been included in confix", err))
+		return nil, err
+	}
+
+	f, err := data.Open(fmt.Sprintf("data/%s", fileName))
```

Comment:
Please see my comment and recommendation to use path/filepath.Join for path portability across Windows and *NIX systems, unless of course this course only runs on *NIX systems?